stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -56,17 +73,38 @@ public ResponseBodyData readFile(String requestId) throws IOException {
       }
       ResponseBodyData bodyData = new ResponseBodyData();
       bodyData.base64Encoded = firstByte != 0;
-      bodyData.data = readContentsAsUTF8(in);
+
+      AsyncPrettyPrinter asyncPrettyPrinter = mRequestIdMap.get(requestId);
+      if (asyncPrettyPrinter != null) {
+        bodyData.data = prettyPrintContentWithTimeOut(asyncPrettyPrinter, in);
```

Comment:
Prefer that you reference the task you created in code so that if someone sees this they can get more context.